docs/src/user_docs/assembly/main.md

```diff
@@ -22,7 +22,7 @@ In this note we use the following terms and notations:
 
 - $p$ is the modulus of the VM's base field which is equal to $2^{64} - 2^{32} + 1$.
 - A *binary* value means a field element which is either $0$ or $1$.
-- Inequality comparisons are assumed to be performed on integer representations of field elements in the range $[0, p)$.
+- Inequality comparisons are assumed to be performed on integer representations of field elements in the range $[0, p]$.
```

Comment:
This is actually correct. The ")" signifies that `p` is not included (i.e., the range is exclusive).

Reply:
Oh yeah, I understood now, fixed.